drivers/hmis/app/models/hmis/hud/processors/current_living_situation_processor.rb

```diff
@@ -6,6 +6,40 @@
 
 module Hmis::Hud::Processors
   class CurrentLivingSituationProcessor < Base
+    def process(field, value)
+      attribute_name = ar_attribute_name(field)
+      current_living_situation = @processor.send(factory_name)
+
+      return super(field, value) unless attribute_name == 'verified_by_project_id'
+
+      # convert HIDDEN to nil
+      verified_by_project_id = value == Base::HIDDEN_FIELD_VALUE ? nil : value
+
+      verified_by_project = Hmis::Hud::Project.find_by(id: verified_by_project_id) if verified_by_project_id
+      # Don't raise if we can't find the project, this can happen with migrated-in values, related to the hack
```

Comment:
`verified_by_project_id` won't be migrated in right?  I'm not sure if we should or shouldn't hide the raise if we can't find the associated project.  If `verified_by_project_id` doesn't have a corresponding project, maybe we should blank it out?

Reply:
Yes that `verified_by_project_id` wouldn't be migrated in, but, see the hack described here: https://github.com/greenriver/hmis-warehouse/pull/4587/files#r1704580947

This was done in order to get the frontend behavior that we want in the form, which is: for migrated-in data, where `VerifiedBy` exists as a string while `verified_by_project_id` is null, the `VerifiedBy` string should be shown in the dropdown (but should not be selectable - only real projects are selectable options).

This means that when a CLS in that migrated-in state is re-submitted without any change to the `verified_by_project_id` dropdown, that string value of `VerifiedBy` is getting re-submitted in the `verified_by_project_id` field. I agree this is pretty confusing! 

I could either:
- Clarify my comments here to better explain what is going on
- Modify the form structure and stop trying to collapse these 2 fields into 1 visually. I think this would result in a less-good UX but clearer code

Reply:
Ah, sorry, I missed that this was the same as `object` so it would potentially be set.  This is good as is  Thanks!